src/nih_wayfinding/app/scripts/views/options-grid/options-grid-partial.html

```diff
@@ -1,5 +1,6 @@
 <div class="options-grid">
```

Comment:
Looks like we're using the inline template in the actual directive. This html partial should just be deleted.